translations/bnpchs-final-2018-chairoikoguma.html

```diff
@@ -0,0 +1,107 @@
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Chairoikoguma (otherwise known as Koguma) is a bear that loves honey a lot. At one time in the honey forest, while wandering around peacefully, Koguma found the 
+  location of the tastiest honey in the world. Koguma wants to stay as long as possible while eating honey.
+</p>
+
+<p style="text-align: justify;">
+  Unfavorably, Jelix, the landlord of the honey forest, despises the honey snatching. The forest populates bees. As soon as Jelix notices Koguma 
```

Comment:
```suggestion
  Unfavorably, Jelix, the owner of the honey forest, despises that his honey is taken. The forest is populated by bees. As soon as Jelix notices Koguma
```

landlord lebih untuk pemilik rumah gitu
trus snatch itu lebih ke copet sih. tapi gw juga kurang tahu vocab yang lebih pas lagi, jadi pakai "taken" dulu aja